Version 1.1.11 (#10172)
ihhub/fheroes2: changelog.txt

version 1.1.11 (14 September 2025)
- update the Russian translation

> - add custom and banned spells selection for the Mage Guild in Editor

Reviewer: ```suggestion
- add custom and banned spells selection for the Mage Guild in the Editor
```

- update the Portuguese translation
- allow to select monsters for Random Monster objects in the Editor

> - expand pop-up message on an object that has an owner in the Editor

Reviewer: ```suggestion
- expand the pop-up message for objects with an owner in the Editor
```

- fix invalid cursor while choosing an object in the Editor
- fix Hero Dialog EXIT button flickering
- fix incorrect sentence for Mines and Observation Tower popup message for OKAY button
- update the Ukrainian translation
- fix logic for hero selection window in the Editor
- add extra checks to avoid corrupted MP2 maps from being loaded
- allow to cast the Summon Boat spell on more tiles
- make the Toolset activities compatible with Android 15 edge-to-edge enforcements
- show the creature image in the "Week of..." and "Month of..." dialog windows
- update the Hungarian translation
- reduce the size of the bundled resurrection.h2d resource file
- update the configuration file to be more human-readable
- fix the shifted by 16 pixels position of a map for the first opening in the Editor
- limit all resolutions to the currently selected resolution in the system in software rendering mode
- bring back compilation of x86_64 for macOS

> - add a pop-up window for clicking on a scenario title in the campaign window

Reviewer: ```suggestion
- add a pop-up window when right-clicking a scenario title in the campaign window
```
